app/controllers/users_controller.rb

```diff
@@ -53,11 +53,18 @@ class UsersController::InvitationsController < Devise::InvitationsController
 
   include ElevatedPermissionsHelper
 
-  def after_invite_path_for(resource)
-    if resource.admin
+  def after_invite_path_for(current_inviter)
+    request_origin = request.referer
+    uri = URI.parse(request_origin)
+    params = CGI.parse(uri.query)
+    role = params['role'].first
+
```

Comment:
Is it not possible to use Rails to read the param e.g. `params[:role]` without having to parse the URL by hand?
http://guides.rubyonrails.org/action_controller_overview.html#parameters

Reply:
+1

Reply:
I`m not accessing to the parameters in the request, I'm accessing to the URL from where the request come from. `Params` only contains the request related parameters not the one I want to access. For this reason I'm parsing the origin URL to be able to access the parameters in the origin in a Ruby way.